api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -211,6 +211,14 @@ message JwtProvider {
   // Specify the clock skew in seconds when verifying JWT time constraint,
   // such as `exp`, and `nbf`. If not specified, default is 60 seconds.
   uint32 clock_skew_seconds = 10;
+
+  // Cache is a LRU cache, its size is specified by jwt_cache_size.
```

Comment:
Why is LRU relevant? Is this an implementation choice we would want to leave flexibility for in different implementations?

Reply:
Based on the gut feeling that most recent used tokens have higher probability of re-occurrence.   That is why LRU purge policy is chosen.

Reply:
I think there is a distinction in xDS APIs between the APIs and client implementations.

For Envoy today, we can pick LRU. Do we also want to say it is mandatory that gRPC picks LRU to manage its cache here? Or that Envoy tomorrow is not allowed to have an alternative implementation?

If the choice of eviction algorithm is really opaque functionally, I'd just not document that this is LRU.

CC @envoyproxy/api-shepherds

Reply:
Hi @htuch I don't quite understand the issue.  This cache is specific to jwt_authn token cache,  not other cache. I am not sure if gRPC is using cache,  but definitely not about gRPC cache.  Not sure why xDS API is relevant here. 
Since the next field is about the cache size.  It is nature for people to ask what happen when the cache reaches its size limit. We need to tell users the purge policy is LRU.

Reply:
These protos are in the xDS API (everything under `api/` is).

I don't understand why we _need_ to tell the users what the purge policy is. The only reason I can imagine this is useful is if it is actionable; would the user act any differently if it was not LRU?

Reply:
OK,  I see.  I am fine with not mentioning the LRU purge policy.

Reply:
Hi @htuch, 
```
Enables JWT cache, its size is specified by jwt_cache_size. Only valid JWT tokens are cached.
```
 Is it okay?

Reply:
Yes, but `*jwt_cache_size*` please.

Reply:
Done.